src/template.rs

```diff
@@ -1,14 +1,22 @@
 use cargo_generate::{GenerateArgs, TemplatePath};
+use reqwest::blocking::Client;
+use serde::Deserialize;
 use std::path::PathBuf;
 
+#[derive(Debug, Deserialize)]
+struct Repository {
+    html_url: String,
```

Comment:
Why not just `url`?

Reply:
I see! Maybe a doc comment about this would be nice, it sounds like something someone else might easily trip over :)

Reply:
Yeah for sure.